.github/workflows/build_and_test.yml

```
@@ -198,6 +198,19 @@ jobs:
         ports:
           - 9411:9411
 
+      redis-auth:
```

```suggestion
      redis-stack:
```
there's already one redis-stack at kong-ee, let's dedup them

---

To dedup we need to change kong-ee redis-stack to use authentication and update the kong-ee tests accordingly to authenticate. Can we keep this one here as a dedicated redis service that provides authentication?

---

in that case we might want to add this one to the local development env https://github.com/Kong/kong/blob/master/scripts/dependency_services/docker-compose-test-services.yml#L40 then name it `redis-auth` too.
not a blocker for this PR, as long as this being done in seperate PR.

---

Done in https://github.com/Kong/kong/pull/13496/commits/c234ce93844ae7ae73123e738bf82ccb4b0907c0